src/BlockTypes/ProductRating.php

```diff
@@ -150,12 +150,7 @@ protected function render( $attributes, $content, $block ) {
 						esc_html( $reviews_count )
 					);
 
-					if ( $is_descendent_of_single_product_block ) {
-						$customer_reviews_count = '<a href="' . esc_url( $product_permalink ) . '">' . $customer_reviews_count . '</a>';
-					} elseif ( $is_descendent_of_single_product_template ) {
-						$product_permalink      = untrailingslashit( $product_permalink );
```

Comment:
I see that the `untrailingslashit()` function call has been removed from the $product_permalink in the new implementation. AFAIK, This function was previously used to remove trailing slashes from the URL. Could you share your thoughts on why it's no longer necessary?

Reply:
I updated the code via [0425ca5](https://github.com/woocommerce/woocommerce-blocks/pull/10040/commits/0425ca52563125a987fe70c0fc111d229931bde4). Could you take another look, please?

Reply:
Thanks for making the changes! 🙌🏻
I still have the same question regarding the `untrailingslashit` function. Do we no longer need it? 🤔

Reply:
No, on the Single Product Block, there is no difference in whether the `/` will be append or not: the interaction will require a full refresh page.

Reply:
AFAIU It's removed from `is_descendent_of_single_product_template`, which doesn't require a full refresh. no? 🤔

Reply:
True!
For the `is_descendent_of_single_product_template` case, now we just append the `#reviews` string (no URL).

Reply:
Although I'm not entirely sure if there are any potential issues that could arise from removing the following code:
```
$product_permalink = untrailingslashit( $product_permalink );
```
I would like to ensure that removing this code won't cause any problems.

Reply:
Okay. It's working as expected, so feel free to remove it. I just wanted to make sure that we don't introduce new errors 🙂

Reply:
Not sure that I'm following you:
- `is_descendent_of_single_product_block` -> we never used `untrailingslashit` 
- `is_descendent_of_single_product_template` -> we used it, but now it isn't necessary because we just append the `#reviews` string. We don't manipulate anymore `$product_permalink` in this case.